docs/post-upgrade-req.asciidoc

```diff
@@ -1,11 +1,10 @@
 [[post-upgrade-req]]
 [role="xpack"]
-= Enable process analyzer after upgrade
+= Enable process analyzer after an upgrade
 
-To enable <<alerts-analyze-events, graphical representations of process relationships>>
-after upgrading to {stack} version 7.9.0 or 7.9.1 from a
-previous minor release (7.8.x, 7.7.x, and so on), you need to update
-`.siem-signals*` system index mappings.
+After upgrading from {stack} version 7.9.x to >= 7.10.x from a previous minor release (e.g., 7.8.x, etc.), you need to update `.siem-signals*` system index mappings to enable <<alerts-analyze-events, graphical representations of process relationships>>.
+
+NOTE: If you are upgrading from a minor release to {stack} version >=7.11.0, there is now a <<signals-migration-api>> that you can use instead of the manual process described below.
```

Comment:
Nit: there is no space between `>=` and `7.11.0` here, but there is one used in the previous sentence.